profiling/build.rs

```diff
@@ -161,6 +166,43 @@ fn generate_bindings(php_config_includes: &str) {
         .expect("bindings to be written successfully");
 }
 
+fn cfg_preload() -> bool {
+    let output = Command::new("php")
+        .arg("-r")
+        .arg("echo (PHP_VERSION_ID >= 70400) ? 1 : 0, PHP_EOL;")
```

Comment:
Is there a specific reason why we invoke php here, instead of php-config? Specifically `php-config --vernum`, which also can be used for `cfg_php_major_version`.

Reply:
No, no reason. I suppose we can change it. Does `php-config` have anything for telling us if it's ZTS or not? IIRC it doesn't, you'd have to parse it from `php -v` or similar.

Reply:
> parse it in rust code instead of bash? 
🤦🏻

Reply:
Fixed, since as you pointed out the `cfg_php_major_version` and `cfg_preload` routines can use the same source.